### ISSUE TITLE
Input tensor size and batch normalization

### ISSUE THREAD

**Issue by axiezai** [axiezai/nl-processors#2]
Need to implement batch normalization and somehow figure how why we are getting 8 features instead of 2 from dataloader.

**Comment by axiezai** [comment]
Still need to match with layer 1 and implement batch normalization

**Comment by axiezai** [comment]
See https://discuss.pytorch.org/t/runtimeerror-expected-object-of-scalar-type-double-but-got-scalar-type-float-for-argument-2-weight/38961/14
